src/components/structures/RoomView.js

```diff
@@ -281,6 +283,7 @@ module.exports = React.createClass({
                         // This is fine: the room just isn't peekable (we assume).
                         this.setState({
                             peekLoading: false,
+                            isPeeking: false,
```

Comment:
I think we also want `isPeeking: false` in the general case of an error when trying to peek (e.g. internal server error), not that it matters hugely, more for consistency.

Reply:
(also note my comment [here](https://github.com/matrix-org/matrix-react-sdk/pull/1463#discussion_r144285513))

Reply:
fixed